lib/data_update_scripts/20200410152018_resync_elasticsearch_documents.rb

```diff
@@ -0,0 +1,75 @@
+module DataUpdateScripts
+  class ResyncElasticsearchDocuments
+    def run
+      # Previous bug: Not getting removed properly
+      sync_docs(Article.pluck(:id), "Article")
+      sync_docs(Comment.pluck(:id), "Comment")
+      sync_docs(User.pluck(:id), "User")
+
+      # Previous bug: Not getting indexed on creation properly
+      index_docs(PodcastEpisode.pluck(:id), "PodcastEpisode")
+      index_docs(Tag.pluck(:id), "Tag")
```

Comment:
should these ids be fetched in batches?

Reply:
Nah, there are only 48k. Its faster to grab them at once then in batches

Reply:
sorry, the comment was applied to all lines, not just tags, but yeah :)